container/core.py

```diff
@@ -731,6 +731,13 @@ def conductorcmd_build(engine_name, project_name, services, cache=True, local_py
                     environment=dict(ANSIBLE_CONTAINER=1)
                 )
 
+                if service.get('build_overrides'):
```

Comment:
I think this actually belongs in `config.py`, and it should be handled as part of the [set_env method](https://github.com/balll/ansible-container/blob/9e2a80a88864a4f66f51283d33905283d10c7735/container/config.py#L116) on the host side. That method would then accept a new `env`  value of `build`, and return the appropriate configuration.